packages/devtools_app/lib/src/memory/memory_instance_tree_view.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../auto_dispose_mixin.dart';
+import '../table.dart';
+import '../table_data.dart';
+import '../utils.dart';
+import 'memory_controller.dart';
+import 'memory_snapshot_models.dart';
+
+class InstanceTreeView extends StatefulWidget {
+  @override
+  InstanceTreeViewState createState() => InstanceTreeViewState();
+}
+
+/// Table of the fields of an instance (type, name and value).
+class InstanceTreeViewState extends State<InstanceTreeView>
+    with AutoDisposeMixin {
+  MemoryController controller;
+
+  final TreeColumnData<FieldReference> treeColumn = _FieldTypeColumn();
+  final List<ColumnData<FieldReference>> columns = [];
+
+  @override
+  void initState() {
+    setupColumns();
+
+    super.initState();
```

Comment:
same as elsewhere, super should be called first for initState.

Reply:
Fixed in all memory classes.